client/src/app/containers/Alert/useAlert.ts

```diff
@@ -0,0 +1,32 @@
+import { nanoid } from '@reduxjs/toolkit';
+import { useDispatch } from 'react-redux';
+
+import { actions } from './slice';
+import { AlertType } from './types';
+
+export interface AlertSetter {
+    (message: string, type: AlertType): string;
+}
+
+interface ReturnType {
+    setAlert: AlertSetter;
+    removeAlert: (id: string) => void;
+}
+
+export function useAlert(timeout = 5000): ReturnType {
```

Comment:
:mag: :book: is `useAlert` a great name for this?

Reply:
`useAlert` is a custom [hook](https://reactjs.org/docs/hooks-intro.html). The convention is that all hooks are named as `use*`.